infrastructure/terraform/per_workspace/modules/etl/sds/main.tf

```diff
@@ -432,3 +440,36 @@ module "bulk_trigger_notification" {
   filter_prefix = "${local.bulk_trigger_prefix}/"
   filter_suffix = ".ldif"
 }
+
+module "proxy_lambda_executor" {
+  source = "./proxy-executor/"
```

Comment:
i'm wondering whether we should give a name that helps the developer pin down where this fits into the ETL. maybe "ETL state lock enforcer" so something like `module etl_state_lock_enforcer` and `source = ./etl-state-lock-enforcer`